complex/ft_domain.c

```diff
@@ -61,7 +61,7 @@ static int ft_open_eq(void)
 		return 0;
 
 	memset(&attr, 0, sizeof attr);
-	attr.wait_obj = FI_WAIT_FD;
+	attr.wait_obj = test_info.eq_wait_obj;
```

Comment:
The code is prepared to handle FI_WAIT_NONE for an EQ.  ft_get_event() needs to change, so that it polls reading the EQ until an event is found.  It currently calls EQ sread, which isn't supported if wait none is specified.